tests/fixtures/requestSchema/serverless.yml

```diff
@@ -0,0 +1,63 @@
+service: service
+
+provider:
+  name: aws
+  apiGateway:
+    requestSchemas:
+      test-model:
+        name: 'TestModel'
+        description: 'Test Description'
+        schema: '${file(dummySchema.json)}'
+      test-model-simple:
+        schema: '${file(dummySchema.json)}'
+      unused-model:
+        name: 'UnusedModel'
+        description: 'I should not be created'
+        schema: '${file(dummySchema.json)}'
+
+functions:
+  target:
+    handler: target.handler
+    events:
+      - http:
+          path: test
+          method: get
+          requestSchema:
+            application/json: test-model
+      - http:
+          path: testSimple
+          method: get
+          requestSchema:
+            application/json: test-model-simple
+      - http:
+          path: test2
+          method: get
+          requestSchema:
+            application/json: '${file(dummySchema.json)}'
+      - http:
+          path: test3
+          method: get
+          requestSchema:
+            application/json:
+              name: TestMethodModel
+              description: 'Test Method Model Desc'
+              schema: '${file(dummySchema.json)}'
+      - http:
+          path: test4
+          method: get
+          request:
+            schema:
+              application/json: '${file(dummySchema.json)}'
+      - http:
+          path: test5
+          method: get
+          requestSchema:
+            application/json: '${file(dummySchema.json)}'
+            text/plain: 'foo'
+      - http:
+          path: test6
```

Comment:
It'll be nice to name paths after what they test, so:

`test` -> `test-model-full`
`testSimple` -> `test-model-simple`
`test2` -> `test-direct-simple`
`test3` -> `test-direct-full`
`test4` -> `test-deprecated-simple`
`test5` -> `test-multiple`
`test6` -> `test-deprecated-multiple`